oml/imce.jpl.nasa.gov/discipline/power/pel.oml

```diff
@@ -0,0 +1,133 @@
+@owl:versionInfo="""$Revision$"""
+@dc:creator="""Flight Systems Engineering, Integration, & Test Section"""
+@dc:date="""$Date$"""
+@dc:type="""Discipline"""
+@dc:title="""PEL Ontology"""
+@dc:description="""The PEL Ontology defines concepts and properties for describing power equipment lists."""
+open terminology <http://imce.jpl.nasa.gov/discipline/power/pel> {
+	
+	extends <http://imce.jpl.nasa.gov/foundation/behavior/behavior>
+	
+	extends <http://iso.org/iso-80000-2>
+	
+	extends <http://iso.org/iso-80000-4>
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Volt Parameter"""
+	concept VoltParameter
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Watt Parameter"""
+	concept WattParameter
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Percent Parameter"""
+	concept ContingencyParameter
+	
+	// TODO Move to a power ontology?
+	@dc:description="""TBD"""
+	@rdfs:label="""Watt Codomain"""
+	concept VoltCodomain
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Watt Codomain"""
+	concept WattCodomain
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Percent Codomain"""
+	concept PowerContingencyCodomain
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""Constant Value Constraint"""
+	concept ConstantValueConstraint
+	
+	@dc:description="""TBD"""
+	@rdfs:label="""MEV Constraint"""
+	concept MEVConstraint
+	
+	VoltParameter extendsConcept behavior:Parameter
+	
+	WattParameter extendsConcept behavior:Parameter
+	
+	ContingencyParameter extendsConcept behavior:Parameter
+	
+	VoltCodomain extendsConcept behavior:ScalarCodomain
+	
+	WattCodomain extendsConcept behavior:ScalarCodomain
+	
+	PowerContingencyCodomain extendsConcept behavior:ScalarCodomain
+	
+	ConstantValueConstraint extendsConcept behavior:Constraint
+	
+	MEVConstraint extendsConcept behavior:Constraint
+	
+	@dc:description="""TBD"""
+	@oml:hasPropertyLabel="""constrainsCBEValue"""
+	@oml:hasReificationLabel="""ConstrainsCBEValue"""
+	reifiedRelationship ConstrainsCBEValue {
+		functional
+		essential
+		unreified = constrainsCBEValue
+		source = MEVConstraint
+		target = behavior:StateVariable }
+		
+	@dc:description="""TBD"""
+	@oml:hasPropertyLabel="""constrainsMEVValue"""
+	@oml:hasReificationLabel="""ConstrainsMEVValue"""
+	reifiedRelationship ConstrainsMEVValue {
+		functional
+		essential
+		unreified = constrainsMEVValue
+		source = MEVConstraint
+		target = behavior:StateVariable }
+		
+	@dc:description="""TBD"""
+	@oml:hasPropertyLabel="""constrainsContingencyValue"""
+	@oml:hasReificationLabel="""ConstrainsContingencyValue"""
+	reifiedRelationship ConstrainsContingencyValue {
+		functional
+		essential
+		unreified = constrainsContingencyValue
+		source = MEVConstraint
```

Comment:
Do you mean ContingencyConstraint here?

Reply:
Same as above - MEVConstraint is what was intended. The constraint is intended to represent some relationship between MEV, CBE and contingency values.

Reply:
Pushed some changes after our discussion today. Please re-review, and resolve if you are satisfied with the changes